src/main/java/org/waarp/openr66/context/R66Session.java

```diff
@@ -705,7 +714,7 @@ public void startup(boolean checkNotExternal) throws OpenR66RunnerErrorException
                 this.businessObject.checkAfterPreCommand(this);
             }
             if (!this.runner.isSender()) {
-                // Check file length according to rank
+                // ConsistencyCheck file length according to rank
```

Comment:
Everytime the word `Check` was used in a comment, it has been replaced with `ConsistencyCheck`